packages/client/bin/cli.ts

```diff
@@ -199,8 +206,68 @@ async function run() {
   // give network id precedence over network name
   const chain = args.networkId ?? args.network ?? 'mainnet'
 
-  // configure common based on args given
+  // configure accounts for mining and prefunding in a local devnet
+  const accounts: [Address, Buffer][] = [] // [address, privateKey]
+  if (args.unlock) {
+    const rl = readline.createInterface({
+      input: process.stdin,
+      output: process.stdout,
+    })
+    const question = (text: string) => {
+      return new Promise<string>((resolve) => {
+        rl.question(text, resolve)
+      })
+    }
+
+    try {
+      for (const addressString of args.unlock) {
+        const address = Address.fromString(addressString)
+        const inputKey = await question(
+          `Please enter the 0x-prefixed private key to unlock ${address}:\n`
+        )
+        const privKey = toBuffer(inputKey)
+        const derivedAddress = new Address(privateToAddress(privKey))
```

Comment:
I know this part is my code but I wanted to update this :)

```suggestion
        const derivedAddress = Address.fromPrivateKey(privKey)
```